lib/omsmailer_web/templates/page/candidate_applied.html.eex

```diff
@@ -0,0 +1,82 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<%= render(OmsmailerWeb.SnippetsView, "head.html", title: "MyAEGEE: New application submitted") %>
```

Comment:
Maybe change this to New candidature submitted? To distinguish event application and position application